supervisor/addons/model.py

```diff
@@ -540,12 +540,12 @@ def _available(self, config) -> bool:
 
         # Home Assistant
         version = config.get(ATTR_HOMEASSISTANT) or self.sys_homeassistant.version
-        if pkg_version.parse(self.sys_homeassistant.version) < pkg_version.parse(
-            version
-        ):
-            return False
-
-        return True
+        return (
+            not version
+            or self.sys_homeassistant.version is None
+            or pkg_version.parse(self.sys_homeassistant.version)
+            >= pkg_version.parse(version)
+        )
```

Comment:
That is very hard to understand. All pythonic in love but it should not end up in Perl. Anyway, this code is currently wrong because of the new version handling, which needs force us to handle exception because a HA version can also be "landingpage" which is not a valid version. Good catch

Reply:
I think that this code can be simplified even more because we default to `self.sys_homeassistant.version` and then check if that is >= that, which is never true


```python
version = config.get(ATTR_HOMEASSISTANT)

if version is None:
    return True

…
```

Are you adding the 2 extra checks because of typing? In that case consider fixing the type of `self.sys_homeassistant.version` or ensure it using an assert, it should always be there?

Reply:
Adjusted, added the Exception handling as well.